server/webhook.go

```diff
@@ -267,6 +268,67 @@ func (p *Plugin) postPullRequestEvent(event *github.PullRequestEvent) {
 	}
 }
 
+func (p *Plugin) handlePRDescriptionMentionNotification(event *github.PullRequestEvent) {
+	action := event.GetAction()
+	if action != "opened" {
+		return
+	}
+
+	body := event.GetPullRequest().GetBody()
+	if strings.Contains(body, "notifications@github.com") {
+		body = strings.Split(body, "\n\nOn")[0]
+	}
+
+	mentionedUsernames := parseGitHubUsernamesFromText(body)
+
+	message, err := renderTemplate("pullRequestMentionNotification", event)
+	if err != nil {
+		mlog.Error("failed to render template", mlog.Err(err))
+		return
+	}
+
+	post := &model.Post{
+		UserId:  p.BotUserID,
+		Message: message,
+		Type:    "custom_git_mention",
+	}
+
+	for _, username := range mentionedUsernames {
+		// Don't notify user of their own comment
+
```

Comment:
Nit: Remove this line
```suggestion
		// Don't notify user of their own comment
```

Reply:
Fixed